### ISSUE TITLE
Identify the images of Cats and Dogs

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/ML-Crate#104]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Cats and Dogs Identification
:red_circle: **Aim** : Create a ML model which will identify the images of the cats and dogs.
:red_circle: **Dataset** : https://www.kaggle.com/chetankv/dogs-cats-images
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

Hello, ML-Crate contributors, this issue is only for the contribution purposes and allocated only to the participants of SWOC 2.0 Open Source Program and JWOC '22 Open Source Program.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.
- This issue is only for 'SWOC' and 'JWOC' contributors of 'ML-Crate' project.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID :
- Approach for this Project :
- What is your participant role?
    - [ ] SWOC 2.0 Participant.
    - [ ] JWOC 2022 Participant.
    - [ ] Contributor

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by VishnuBhaarath** [comment]
Hi,
 I have registered for JWOC 22 can i work on this, my approach would be using CNN to classify the images.

**Comment by abhisheks008** [comment]
Follow the issue guidelines and mention the details properly @VishnuBhaarath

**Comment by VishnuBhaarath** [comment]
> Follow the issue guidelines and mention the details properly @VishnuBhaarath

Hi
Full Name: Vishnu Bhaarath
Github profile link:https://github.com/VishnuBhaarath
Project title: cats and dog identification
Approach for this project: Have gone through the dataset, it consist of different folders for training as well as testing with images the labels are dogs and cats, so my approach would be image preproccessing, then will be applying image augumentation techniques and finally will be setting up Convolutional Neural network (CNN) for training purpose, will be applying dropout to avoid overfitting, and will be predicting the results on testing dataset with metrics such as accuracy and f1 score.
Can I work on this issue

**Comment by VishnuBhaarath** [comment]
> > Follow the issue guidelines and mention the details properly @VishnuBhaarath
> 
> Hi Full Name: Vishnu Bhaarath Github profile link:https://github.com/VishnuBhaarath Project title: cats and dog identification Approach for this project: Have gone through the dataset, it consist of different folders for training as well as testing with images the labels are dogs and cats, so my approach would be image preproccessing, then will be applying image augumentation techniques and finally will be setting up Convolutional Neural network (CNN) for training purpose, will be applying dropout to avoid overfitting, and will be predicting the results on testing dataset with metrics such as accuracy and f1 score. Can I work on this issue

JWOC 22 participant

**Comment by abhisheks008** [comment]
Issue assigned to you @VishnuBhaarath. Go ahead.

**Comment by abhisheks008** [comment]
What's the update here @VishnuBhaarath

**Comment by VishnuBhaarath** [comment]
> What's the update here @VishnuBhaarath

Hi @abhisheks008  will be giving a PR by today before EOD

**Comment by abhisheks008** [comment]
Great! Waiting for your PR. Thanks for the update @VishnuBhaarath

**Comment by VishnuBhaarath** [comment]
> Great! Waiting for your PR. Thanks for the update @VishnuBhaarath

Hi @abhisheks008  have given the PR thanks for your patience.
